src/Core/Utilities/SecretsManagerPlanStore.cs

```diff
@@ -0,0 +1,181 @@
+﻿using Bit.Core.Enums;
+using Bit.Core.Models.StaticStore;
+
+namespace Bit.Core.Utilities;
+
+public static class SecretsManagerPlanStore
+{
+    public static IEnumerable<Plan> CreatePlan()
+    {
+        return new List<Plan>
+        {
+            new Plan
+            {
+                Type = PlanType.EnterpriseMonthly,
+                Product = ProductType.Enterprise,
+                BitwardenProduct = BitwardenProductType.SecretsManager,
+                Name = "Enterprise (Monthly)",
+                NameLocalizationKey = "planNameEnterprise",
+                DescriptionLocalizationKey = "planDescEnterprise",
+                CanBeUsedByBusiness = true,
+                BaseSeats = 0,
+                BaseServiceAccount = 200,
+                HasAdditionalSeatsOption = true,
+                HasAdditionalServiceAccountOption = true,
+                TrialPeriodDays = 7,
+                HasPolicies = true,
+                HasGroups = true,
+                HasDirectory = true,
+                HasEvents = true,
+                HasTotp = true,
+                Has2fa = true,
+                HasApi = true,
+                HasSelfHost = true,
+                HasSso = true,
+                HasKeyConnector = true,
+                HasScim = true,
+                HasResetPassword = true,
+                UsersGetPremium = true,
+                HasCustomPermissions = true,
+                UpgradeSortOrder = 3,
+                DisplaySortOrder = 3,
+                StripeSeatPlanId = "sm-enterprise-seat-monthly",
+                StripeStoragePlanId = "service-account-monthly",
```

Comment:
Should be `StripeServiceAccountPlanId`. Not sure if you're referencing this elsewhere so I'll leave you to make this change.

Reply:
Thanks for pointing this out. it has been renamed properly